config/indexer_settings_dev.yml

```diff
@@ -2,6 +2,7 @@
 
 processing_thread_pool: 5
 hathi_overlap_path: ignorethis_hathi/
-hathi_overlap_file: final_hathi_overlap.csv
+hathi_mono_overlap_file: hathi_filtered_by_mono_overlap.csv
+hathi_multi_overlap_file: hathi_filtered_by_multi_overlap.csv
```

Comment:
thinking one file with `ht_id`, `ht_bib_key` and `ht_access` (for deny/allow) would be everything we need.

All multis will be linked to `ht_bib_key` in blackcat, but, we'll still need deny/allow to inform language of the link.

Reply:
https://github.com/psu-libraries/psulib_traject/pull/215#discussion_r441220566 moving on with two file for now